### PR TITLE
Fix/login form

### PR DESCRIPTION
**Pull request by AnastasiaLes**
1. Fixed styled 
2. Deleted Formik use React Hook Form
3. Added validation and error messages
Screenshot 2022-11-08 220035

Screenshot 2022-11-08 223532

### CODE REVIEW

**Comment by lyudad** [comment]
it look super nice. but why buttons have different colors?

File: apps/freelance/src/pages/SignupPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { LoginForm } from '@freelance/components';
+import { Wrapper} from './styles';
+
+import { useTranslation } from 'react-i18next';
+
+export default function LoginPagePage() {
+  const { t } = useTranslation();
+ 
+
+
```
Comment (lyudad) [inline apps/freelance/src/pages/SignupPage/index.tsx:11]: delete extra space. I think eslint for you is not working. please double check

**Comment by AnastasiaLes** [comment]
@lyudad  I added hover, so it's a little darker

File: libs/components/src/lib/login-form/login-form.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+const schema: yup.SchemaOf<Partial<FormValues>> = yup.object({
+  email: yup.string().required(),
+  password: yup.string().required(),
+});
+
+export function LoginForm() {
+  const { t } = useTranslation();
+  const [form] = Form.useForm();
+  const { register, handleSubmit, formState } = useForm<FormValues>({
```
Comment (lyudad) [inline libs/components/src/lib/login-form/login-form.tsx:24]: hmm I think its my fault I forget to say. Please add this rule
Снимок экрана 2022-11-08 в 21 45 29

File: libs/components/src/lib/login-form/styles.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+`;
+
+export const StyledInput = styled.input`
+  width: 100%;
+  padding: 10px;
+  border-radius: 5px;
+  border-color: grey;
+
+  :hover{
+    border-color: #3944BC;
```
Comment (lyudad) [inline libs/components/src/lib/login-form/styles.ts:17]: all colours should be in
Снимок экрана 2022-11-08 в 21 47 06
 constants

File: apps/freelance/src/pages/SignupPage/index.tsx (hunk 1 above)
Comment (AnastasiaLes) [reply apps/freelance/src/pages/SignupPage/index.tsx:11]: deleted

File: libs/components/src/lib/login-form/styles.ts (hunk 3 above)
Comment (AnastasiaLes) [reply libs/components/src/lib/login-form/styles.ts:17]: @lyudad   fixed
image

**Pull request merged**